.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: ruby
+cache:
+  bundler: true
+  npm: true
```

Comment:
Travis already caches NPM install / node modules automatically, but I've made this explicit in the config so readers don't need to know about this default, and to maintain parity with the bundler caching.